packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.css

```diff
@@ -87,34 +105,34 @@
   );
 }
 
+.selectedOption {
+  border-radius: var(--radius-large);
+  background-color: var(--color-surface--background);
+}
+
 .actions::before {
   content: "";
   display: block;
   position: absolute;
   top: calc(var(--space-base) * -1);
   right: 0;
   left: 0;
-  width: 100%;
   height: var(--space-base);
-  background-image: linear-gradient(
-    to bottom,
-    rgba(255, 255, 255, 0),
-    rgba(255, 255, 255, 1)
+  background: linear-gradient(
+    180deg,
+    rgba(255, 255, 255, 0) 0%,
```

Comment:
Unrelated and not a fast-follow as it's more of a "should we do this", but can/should we codify this gradient styling? We're using it all over the place in ComboBox and DataList

Reply:
Note for me: we should also have guidance around when to use the fade

Reply:
For anyone reading, we had a quick convo about this in Office Hours today.  We'll look to extract this in someway - Chris, would you like me to put some sort of a ticket for this consideration in the backlog?

Reply:
I think that makes sense, thanks!